src/Command/TailwindBuildCommand.php

```diff
@@ -33,6 +33,7 @@ protected function configure(): void
     {
         $this
             ->addOption('watch', 'w', null, 'Watch for changes and rebuild automatically')
+            ->addOption('poll', null, null, 'Apply --poll flag to the command')
```

Comment:
Let's be more descriptive on what it does, how about:
```suggestion
            ->addOption('poll', null, null, 'Use polling instead of filesystem events when watching')
```

Reply:
Indeed, your description is far more relevant